tests/integration/study_data_blueprint/test_binding_constraints.py

```diff
@@ -784,17 +772,44 @@ def test_for_version_870(self, client: TestClient, user_access_token: str, study
 
         # Check that the matrices are daily/weekly matrices
         expected_matrix = np.zeros((366, 1))
-        for term_alias in ["lt", "gt", "eq"]:
-            res = client.get(
-                f"/v1/studies/{study_id}/raw",
-                params={
-                    "path": f"input/bindingconstraints/{bc_id_w_matrix}_{term_alias}",
-                    "depth": 1,
-                    "formatted": True,
-                },  # type: ignore
-            )
+        for operator in ["less", "equal", "greater", "both"]:
+            if operator != "both":
+                res = client.put(
+                    f"/v1/studies/{study_id}/bindingconstraints/{bc_id_w_matrix}",
+                    json={"operator": operator, f"{operator}_term_matrix": expected_matrix.tolist()},
+                )
+            else:
+                res = client.put(
+                    f"/v1/studies/{study_id}/bindingconstraints/{bc_id_w_matrix}",
+                    json={
+                        "operator": operator,
+                        "greater_term_matrix": expected_matrix.tolist(),
+                        "less_term_matrix": expected_matrix.tolist(),
+                    },
+                )
             assert res.status_code == 200, res.json()
```

Comment:
All combinations of ("old operator", "new operator") are not tested, it seems we need to add some.
Also, it would be better to check that the content is correctly copied with something else than zeros, which could be some kind of default values.

Reply:
note: those combinations could be tested in unit tests, not in integration tests, instead